packages/aragon-api/src/index.js

```diff
@@ -139,6 +126,34 @@ export class AppProxy {
     )
   }
 
+  /**
+   * Get current path for the app. Useful for in-app routing and navigation.
+   *
+   * @return {Observable} Multi-emission Observable that emits the app's current path every time a change is detected.
+   */
+  paths () {
```

Comment:
Shouldn’t it be named `path`? Even if it updates, it’s always a single path that is being received.

Reply:
Hmm yes, we do name `network` as singular even though you're observing the network changes over time. Let's rename it back to `path()`.

`getApps()` should be named `apps` or `installedApps`, but is a bit problematic due to backwards compatibility issues. I'll add `installedApps` and deprecate `getApps()`.

Reply:
Changes for `installedApps()` and `currentPath()` are in https://github.com/aragon/aragon.js/pull/374.

`paths()` has been renamed to `path()` in 1e251f1.